test/handler/test_solhint_handler.vader

```diff
@@ -51,15 +51,14 @@ Execute(The solhint handler should parse linter error messages correctly):
   \   },
   \ ],
   \ ale#handlers#solhint#Handle(bufnr(''), [
-  \   'contracts/Bounty.sol: line 1, col 17, Warning - Compiler version must be fixed (compiler-fixed)',
```

Comment:
Instead of modifying these tests add new ones with the new pattern.  ALE should be able to parse and report both the older and newer patterns.